examples/live/check_results.py

```diff
@@ -1,208 +1,236 @@
-            log.error('Coincident Trigger Test Failed')
-      return coinc_fail
+        if verbose:
+            log.info('IFO SNRs: '+str(snr_list))
+            log.info('Network SNR: %f', network_snr)
+            log.info('IFO End Time: %f', trig_props['tc'][x]) 
+            log.info('Mass 1: %f', trig_props['mass1'][x])
+            log.info('Mass 2: %f', trig_props['mass2'][x])
+            log.info('Spin1z: %f', trig_props['spin1z'][x])
+            log.info('Spin2z: %f', trig_props['spin2z'][x])
       
 
+    # check if injections match trigger params
+    for i in range(n_injs):
+        has_match = False
+        for j in range(n_coincs):
+            if close(inj_time[i],trig_props['tc'][j],1.0):
+                match = True
+                if not close(inj_mass1[i], trig_props['mass1'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i mass1 does not match simultaneous trigger's mass1",i)
+                if not close(inj_mass2[i], trig_props['mass2'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i mass2 does not match simultaneous trigger's mass2",i)
+                if not close(inj_spin1z[i], trig_props['spin1z'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i spin1z does not match simultaneous trigger's spin1z",i)
+                if not close(inj_spin2z[i], trig_props['spin2z'][j], 5e-7):
+                    match = False
+                    log.error("Injection %i spin2z does not match simultaneous trigger's spin2z",i)
+                if not close(15.0, trig_props['net_snr'][j], 0.05):
+                    match = False
+                    log.error("Injection %i network snr does not match simultaneous trigger's network snr", i)
+                has_match = match
+        
+        if not has_match:
+            coinc_fail = True
+            log.error('Injection %i has no match',i)
+            
+    if coinc_fail:
+        log.error('Coincident Trigger Test Failed')
+    return coinc_fail
+
+      
+verbose = False
 log.basicConfig(level=log.INFO, format='%(asctime)s %(message)s')
 sim_gps_start = 1272790000
-sim_gps_end =  1272790500
+sim_gps_end =  1272790300
 sim_f_lower = 18
 fail = False
 lsctables.use_in(LIGOLWContentHandler)
-      
-inj_xmls = sorted(glob.glob('./hwinjcbc*.xml.gz'))
-if len(inj_xmls)==0:
-      log.error('No inj_xml found when checking results')
-      fail=True
+
+if not os.path.exists('./test_inj.hdf'):
+    log.error('No injection file found when checking results')
+    fail=True
 else:
-      path_to_inj_xml = inj_xmls[0]
-      log.info('injection xml found')
-      inj_doc = ligolw_utils.load_filename(
-            path_to_inj_xml, False, contenthandler=LIGOLWContentHandler)
-      inj_table = lsctables.SnglInspiralTable.get_table(inj_doc)
-   
-      tested_detectors = {'H1', 'L1', 'V1'}
+    if verbose:
+         log.info('injection hdf found')
```

Comment:
I do not think this block is necessary. If `test_inj.hdf` is missing, the code that tries to read it will just fail with a pretty obvious exception.